### ISSUE TITLE
Remove Makefiles from tables folder

### ISSUE THREAD

**Issue by reiner-dolp** [liblouis/js-build#6]
See discussion in original emscripten pull request

**Comment by reiner-dolp** [comment]
```
# we are in a docker container, pwd is always clean...
# rm -rf ./out-emscripten-install
emmake make install prefix="$(pwd)/out-emscripten-install"
cp -R ./out-emscripten-install/share/liblouis/tables/ ../js-build/tables/
`

**Comment by reiner-dolp** [comment]
Implemented in liblouis/liblouis#327
